frontend/components/commons/forms/questions-form/QuestionsForm.component.vue

```diff
@@ -50,20 +50,22 @@
     </div>
     <div class="footer-form">
       <div class="footer-form__left-footer">
-        <BaseButton
-          type="button"
-          ref="clearButton"
-          class="primary outline small"
-          @click.prevent="onClear"
-        >
-          <span v-text="'Clear'" />
-        </BaseButton>
+        <div class="buttons-area">
+          <BaseButton
+            type="button"
+            ref="clearButton"
+            class="primary text"
+            @click.prevent="onClear"
+          >
+            <span v-text="'Clear'" />
+          </BaseButton>
+        </div>
```

Comment:
maybe it's a merging problem, but the div.buttons-area should not be here

Reply:
My bad, I have included again after resolve conflict